### DISCUSSION TITLE
cannot use parent in class with no parent

### DISCUSSION THREAD

**Discussion by adityadees** [laravel/nova-issues#4270] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4
- PHP Version: 8
- Database Driver & Version:
- Operating System and Version: windows 11
- Browser type and version:chrome
- Reproduction Repository: https://github.com/###/###

### Description:
on vscode maybe cause php intelephense they say

![image](https://user-images.githubusercontent.com/37553901/168941552-229b9f8c-18a1-441c-8d4c-5d3d10f5f22c.png)

cannot use parent in class with no parent inside `app/nova/resource.php`

**Comment by crynobone** [comment, marked as answer]
`App\Nova\Resource` extends `Laravel\Nova\Resource` which contains traits that provide these methods
